app/components/atoms/socialMediaButtons/socialMediaButton.component.tsx

```diff
@@ -0,0 +1,77 @@
+import {FC} from 'react';
+import {ButtonProps, StyledButton} from './socialMediaButton.styles';
+
+export const Button: FC<ButtonProps> = ({socialMediaIcon, ...props}) => {
+  let icon;
+  switch (socialMediaIcon) {
+    case 'Instagram':
+      icon = (
+        <a href="https://www.instagram.com/cultoperrocafe/" target="_blank">
+          <div className="socialMediaButton">
+            <svg
+              className="socialMediaIcon"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 19.13 19.13"
+            >
+              <g id="diseño">
```

Comment:
I don't dislike naming stuff in Spanish when coding, but mixing languages it's a no go. It's better to keep it in the main language in the code (English).

Reply:
I can speak for this!

I believe the SVG file was taken from the Figma file. Some SVG elements will have Spanish words from the Adobe Illustrator project (made by Angel).

I'd suggest disregarding the content of these SVG files. We can optimize them later.